snap/snapcraft.yaml

```diff
@@ -425,9 +425,6 @@ parts:
       ./kubectl completion bash | sed "s/complete -o default -F __start_kubectl kubectl/complete -o default -F __start_kubectl microk8s kubectl/g" | sed "s/complete -o default -o nospace -F __start_kubectl kubectl/complete -o default -o nospace -F __start_kubectl kubectl/g" > kubectl.bash
       ./kubectl completion bash | sed "s/complete -o default -F __start_kubectl kubectl/complete -o default -F __start_kubectl microk8s.kubectl/g" | sed "s/complete -o default -o nospace -F __start_kubectl kubectl/complete -o default -o nospace -F __start_kubectl kubectl/g" > kubectl.bash
 
-      if $REMOVE_KUBE_SNAP_BINS; then
```

Comment:
@ktsakalozos: i'm not sure what this code does exactly, but it relies on an env var that the new `wrapper.sh` script creates now in a different part. I removed it for now to make the build work, but is it still necessary?